geode-docs/tools_modules/gfsh/command-pages/connect.html.md.erb

```diff
@@ -139,6 +140,13 @@ when connecting to the JMX manager.
 <li>If the parameter is specified without a value: <code class="ph codeph">true</code></li>
 </ul></td>
 </tr>
+<tr class="odd">
+<td><span class="keyword parmname">\-\-skip-ssl-validation</span></td>
+<td>When SSL communication is enabled and this option is specified or assigned the value <code>true</code>, authentication accepts all SSL certificates. This option exists to facilitate testing, and it is not intended for production systems.</td>
```

Comment:
do we want to be more specific in saying that this option actually only allows client to accept all certificates presented by the server, but the server will still need to validate the client's certificate.